compiler-plugin-tests/src/test/resources/sample_package_11/main.bal

```diff
@@ -0,0 +1,13 @@
+import ballerinax/transformer as _;
+
+public isolated function helloWorld(string firstName) returns string => firstName;
+
+public function helloWorld1(table<map<int>> lastName) => ();
+
+public function helloWorld2(string... names) returns string => "Hello World";
+
+public isolated function helloWorld3(string firstName, string lastName = "Root") returns string => "Hello World";
+
+type Annot record {
+    string val;
+};
```

Comment:
Shall we have a test case to check the behaviour when the package is having ONLY public expression bodied functions?